pr-data.csv

```diff
@@ -640,9 +640,9 @@ https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d6
 https://github.com/apache/shardingsphere-elasticjob,bdfcaff0c1a702c3ecb44adf46d609a3f0e86c5e,elasticjob-lite/elasticjob-lite-core,org.apache.shardingsphere.elasticjob.lite.internal.snapshot.SnapshotServiceEnableTest.assertMonitorWithCommand,OD,,,https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/78
 https://github.com/apache/skywalking,edf0bce37f682c093d1ea5bcca0a6d2d1ecc2cc4,apm-commons/apm-datacarrier,org.apache.skywalking.apm.commons.datacarrier.consumer.testConsumerLessThanChannel,OD,Accepted,https://github.com/apache/skywalking/pull/5846,
 https://github.com/apache/skywalking,edf0bce37f682c093d1ea5bcca0a6d2d1ecc2cc4,apm-commons/apm-datacarrier,org.apache.skywalking.apm.commons.datacarrier.consumer.testConsumerMoreThanChannel,OD,Accepted,https://github.com/apache/skywalking/pull/5846,
-https://github.com/Apache/Struts,3a7c35a014f8a3471a1b75e7546e01ecf21d5db3,core,com.opensymphony.xwork2.validator.AnnotationActionValidatorManagerTest.testGetValidatorsForInterface,ID,Opened,https://github.com/apache/struts/pull/458,
-https://github.com/Apache/Struts,3a7c35a014f8a3471a1b75e7546e01ecf21d5db3,core,com.opensymphony.xwork2.validator.AnnotationActionValidatorManagerTest.testSkipUserMarkerActionLevelShortCircuit,ID,Opened,https://github.com/apache/struts/pull/458,
-https://github.com/Apache/Struts,13d9053050c9e4fb2ef049db6a37d3f6eebf48fa,plugins/json,org.apache.struts2.json.JSONValidationInterceptorTest.testValidationFails,ID,Opened,https://github.com/apache/struts/pull/457,
+https://github.com/Apache/Struts,3a7c35a014f8a3471a1b75e7546e01ecf21d5db3,core,com.opensymphony.xwork2.validator.AnnotationActionValidatorManagerTest.testGetValidatorsForInterface,ID,Accepted,https://github.com/apache/struts/pull/458,
+https://github.com/Apache/Struts,3a7c35a014f8a3471a1b75e7546e01ecf21d5db3,core,com.opensymphony.xwork2.validator.AnnotationActionValidatorManagerTest.testSkipUserMarkerActionLevelShortCircuit,ID,Accepted,https://github.com/apache/struts/pull/458,
```

Comment:
The following PRs are Closed as developers seem to be against code changes to fix ID tests 
https://github.com/apache/struts/pull/458
https://github.com/apache/struts/pull/457